ChangeLog.md

```diff
@@ -18,12 +18,15 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
-3.1.4
+3.1.5
 -----
 - Emscripten no longer uses the `allocate()` runtime function.  For backwards
```

Comment:
I assume this change actually didn't make the cut?

Reply:
Correct, this change (https://github.com/emscripten-core/emscripten/commit/6f0307262b4ee9778a32bcb20e9085372e19aa4a) landed today, but the released change is from Friday.